runners/spark/src/main/java/org/apache/beam/runners/spark/io/SourceRDD.java

```diff
@@ -216,8 +216,10 @@ public int hashCode() {
     private final MicrobatchSource<T, CheckpointMarkT> microbatchSource;
     private final SparkRuntimeContext runtimeContext;
 
+    // to satisfy Scala API.
```

Comment:
I see from browsing the code that this is verbatim from elsewhere in the file, so disregard this comment. It is status quo. Nonetheless, I think it might be worth noting that it is probably fixed in 2.11.5 and greater, so maybe also 2.10.6 or greater (I didn't decompile to find out). We use 2.10.5, which I assume might have to do with the Spark deps.

CC: @amitsela